fetch_data.py

```diff
@@ -547,15 +547,15 @@ def csv_metadata(place_name, place_id, place_type, lat=None, lon=None):
         # if lat and lon and type huc12, then it's a local / point-to-huc query
         if place_type == "huc12":
             metadata += "# Corresponding HUC12 code: " + place_id + "\n"
-    elif place_type == "communities":
+    elif place_type == "community":
```

Comment:
This was a change that came about from my modification of this function: https://github.com/ua-snap/data-api/blob/2f9ff0925f6fc63782bd83021ce4ad0a1ec92c2b/validate_data.py#L193

Specifically, in that function I request from Geoserver the corresponding place type and rather than have a lookup table for "'community': 'communities'" for example, I simply changed the communities to community so the type coming from Geoserver was what dictated the CSV metadata being generated.

As for the HUC12 vs community stuff, this is true for the ALFRESCO veg and flam, but I don't know if it's critically important to make that distinction in the metadata of the returned CSV for a community. I'm open to the change if determined to be necessary, but it isn't in place in the production application, so this should probably be done in a future feature branch if we want to make that change.